contracts/PodCastNFT.sol

```diff
@@ -127,42 +139,8 @@ contract PodCastNFT is ERC721URIStorage, Ownable {
         string[] memory _roles,
         string memory _point
     ) internal view returns (string memory) {
-        string memory _name = "Membership NFT";
-        string
-            memory _description = "The membership card of this Henkaku community represents the contribution of the podcast.\\n\\n"
-            "**Special thanks**\\n\\n"
-            "NFT Design:\\n\\n"
-            "Digital Garage team\\n\\n"
-            "Yukinori Hidaka, Saoti Yamaguchi, Masaaki Tsuji, Yuki Sakai, Yuko Hidaka, Masako Inoue, Nanami Nishio, Ruca Takei, Ryu Hayashi.\\n\\n"
-            "Engineering:\\n\\n"
-            "isbtty, yasek, geeknees";
-
-        bytes memory _attributes;
-        // TODO implement roles
-        _attributes = abi.encodePacked(
-            '"attributes": [{"trait_type": "Role", "value": "',
-            '"},',
-            '{"display_type": "number", "trait_type": "Point", "value": "',
-            _point,
-            '"}]'
-        );
-
-        string memory json = Base64.encode(
-            abi.encodePacked(
-                '{"name": "',
-                _name,
-                '",'
-                '"description": "',
-                _description,
-                '",'
-                '"image": "',
-                _imageURI,
-                '",',
-                string(_attributes),
-                "}"
-            )
-        );
-        return string(abi.encodePacked("data:application/json;base64,", json));
+        // TODO
```

Comment:
this is temp solution to avoid tx limitation error 😭

Reply:
What exact error did you get?